server/command.go

```diff
@@ -517,9 +480,14 @@ func executeUninstallServer(p *Plugin, c *plugin.Context, header *model.CommandA
 		&model.WebsocketBroadcast{},
 	)
 
-	const uninstallInstructions = `Jira instance successfully disconnected. Go to **Settings > Applications > Application Links** to remove the application in your Jira Server or Data Center instance.`
+	var uninstallInstructions string
+	if instanceType == "cloud" {
+		uninstallInstructions = `Jira instance successfully disconnected. Go to [**Settings > Apps > Manage Apps**](%s/plugins/servlet/upm) to remove the application in your Jira Cloud instance.`
```

Comment:
0/5 consider adding `GetManageAppsURL()` to `ji` and just use the method here?

Reply:
Yeah, that makes sense. :+1: